### ISSUE TITLE
Number of captcha's skyrocketed since update

### ISSUE THREAD

**Issue by KevinNoordover** [PokemonGoMap/PokemonGo-Map#1599]
Since I have updated my map to the latest commit, the number of captcha's I am seeing has tripled. With the commit I downloaded on 19 november, I got a decent amount of captcha's, but with the commit I downloaded yesterday things went downhill, while I expected it to reduce the number of captcha's.

Anybody else noticing this trend?

**Comment by gonjiak** [comment]
All changes made relevant for captchas are more likely to reduce them, not vice versa. Using the current branch, you should also use the following PRs: #1530 #1476 #1591

Use the newly implemented beehive mode (preferable step-limit 2 or 3), gather as much spawn point information as possible to switch to spawnpoint scanning. With this, captchas shouldn't bother much anymore.

**Comment by KevinNoordover** [comment]
I was already using the alternative beehive before the update, using the pull request version. I belive that may be the reason that others do experience less captcha's, while the opposite is true for me.

I will try the PRs you mentioned later today, to see if it makes a difference

**Comment by hayelf** [comment]
Go for #1588 instead of #1476 . I heard 1476 is in a conflicted state and 1588 has worked very well for me

**Comment by omgms** [comment]
#1476 has been re-based

**Comment by ultrafunkamsterdam** [comment]
@gonjiak Sorry for the n0b question, but how can i apply uncommited pr's like #1530 #1476 #1591  to my fork ?

**Comment by gonjiak** [comment]
@ultrafunkamsterdam Well, I am not familiar with Github too and did it by modifying the code manually, but here is what another user wrote at reddit:

If there is anyone new to all of this, this is how you would install that:
1. Right click into your pokemon go map folder and choose "Git Bash Here"
2. Enter "git fetch origin pull/**ID**/head:PR#-name" into the Git Bash. Without quotes and replacing **ID** with the pull number, and replacing PR#-name with what you would like to call it. EX. git fetch origin pull/1367/head:1367_Beehive
3. Now enter "git checkout PR#-name" (Without quotes and replacing PR#-name with the same name as in step 2. EX. git checkout 1367_Beehive
Your all set, enjoy!!

**Comment by ultrafunkamsterdam** [comment]
Awesome, thanks for your quick response 
will try immediately!

**Comment by KevinNoordover** [comment]
You have to merge them to do that.

Anyway, after a night scanning there was no improvement at all, the avarage per hour is slightly higher. To completly rule out this update, I'll create new accounts. The thing I tried to avoid.

**Comment by klvs** [comment]
@KevinNoordover I've been facing the same issue. Running a large (city wide) scan is fairly captcha heavy. I'm wondering if you've found any solution or just threw in the towel? 

Did you find that account creation netted fewer captchas / hr than solving them? I think a lot of folks here are running small scans and thus don't mind having to solve a few captchas for their 2-5 workers. I'm covering a much larger area and it quickly becomes too expensive.

**Comment by KevinNoordover** [comment]
@klvs I came to the conclusion that using the 0.45 api is the reason and that the only way to stop captcha's was to stop scanning. Nothing what I did reduced it.

**Comment by ultrafunkamsterdam** [comment]
i run amsterdam complete and got around 8-13 an hour

**Comment by klvs** [comment]
@ultrafunkamsterdam per account or total? I've been doing San Francisco proper and was blowing through way more than 8-13 / hr. It could be just a location related thing, but regardless what settings are you running it on?

**Comment by ultrafunkamsterdam** [comment]
san fransisco is 10x amsterdam lol


Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Alex Roman <notifications@github.com> Date: 12/12/16  04:10  (GMT+01:00) To: PokemonGoMap/PokemonGo-Map <PokemonGo-Map@noreply.github.com> Cc: Leon <leon@ultrafunk.nl>, Mention <mention@noreply.github.com> Subject: Re: [PokemonGoMap/PokemonGo-Map] Number of captcha's skyrocketed since update (#1599) 
@ultrafunkamsterdam per account or total? I've been doing San Francisco proper and was blowing through way more than 8-13 / hr. It could be just a location related thing, but regardless what settings are you running it on?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/PokemonGoMap/PokemonGo-Map","title":"PokemonGoMap/PokemonGo-Map","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/PokemonGoMap/PokemonGo-Map"}},"updates":{"snippets":[{"icon":"PERSON","message":"@klvs in #1599: @ultrafunkamsterdam per account or total? I've been doing San Francisco proper and was blowing through way more than 8-13 / hr. It could be just a location related thing, but regardless what settings are you running it on? "}],"action":{"name":"View Issue","url":"https://github.com/PokemonGoMap/PokemonGo-Map/issues/1599#issuecomment-266333996"}}}

**Comment by klvs** [comment]
Right, sure, but even with 10ish workers we'd get hit pretty hard with captchas...

**Comment by Thyllz** [comment]
I'm doing a much smaller area than Amsterdamn and I also get more than 8-13/h

**Comment by ultrafunkamsterdam** [comment]
it all hangs together with your settings i suppose. i use a low  -asi 1800 and -ari 600 and for workers i have a different formula. ((st * st × 3) / 20) + 2 and round up to top. for accounts i do workers * 1.5
this makes sure the accounts get their rest and there are anough accounts for the workers. also keep -sd high (15-18) and kph to 23-25. -ld to 10 -mf 5    . if you use proxies, switch them often (i switch once a minute). oh,  i dont cover lakes or empty land, and all accounts have tutorial completed and are min level 5 (although i dont know if it has any impact at all. play with your settins, or try mine. hopefully it helps. i turned everythibg off now because of the special week coming (where niantic dont want any maps ofcourse, so playing using 0 45 could result in a permban. be carefull

**Comment by sebastienvercammen** [comment]
With all of our major latest changes (an immense amount, if I say so myself), this "issue" is no longer relevant. Any commits have only decreased the captcha rate.

Our current test setup ran for 15 hours without any captchas, although this did include some features that will be polished and merged in the upcoming weeks. Still, even without those, captcha rate is considerably lower.
